plugins/git/commands

```diff
@@ -6,14 +6,13 @@ git_archive_all() {
   TMP_WORK_DIR=$(mktemp -d)
   chmod 755 $TMP_WORK_DIR
   unset GIT_DIR GIT_WORK_TREE
-  git clone -q $DOKKU_ROOT/$APP $TMP_WORK_DIR &> /dev/null
   pushd $TMP_WORK_DIR > /dev/null
+  git clone -q $DOKKU_ROOT/$APP $TMP_WORK_DIR &> /dev/null
   git config advice.detachedHead false
   git checkout $REV &> /dev/null
   git submodule update --init --recursive &> /dev/null
   find -name .git -prune -exec rm -rf {} \; > /dev/null
   tar c .
-  popd > /dev/null
```

Comment:
When running locally from a directory that the `dokku` user didn't have access to, it would cause the deployment to fail because of `pipefail`. Additionally, it was unnecessary as it was being executed in a subshell anyway.